Gifski/MainWindowController.swift

```diff
@@ -214,6 +215,8 @@ final class MainWindowController: NSWindowController {
 			if $0 == .OK {
 				self.startConversion(inputUrl: inputUrl, outputUrl: panel.url!)
 			}
+
+			self.accessoryViewController = nil
```

Comment:
When I fixed retain cycles in the save panel, it turned out that no-one was keeping the reference to the it, so it was alive just because we had retain cycles. We only pass the view to the `accessoryView` so someone has to hold the controller itself. And because of that we should nil it when not using.

Reply:
Ah, ok. Makes sense.